### DISCUSSION TITLE
Collapsible page groups

### DISCUSSION THREAD

**Discussion by emmerich** [GitbookIO/community#15] [Feature Requests]
### What's your feature/idea?

I'd like page groups to be collapsible. 

### What's your desired outcome?

If I create a page with children, I can collapse the parent page:

![Screenshot 2023-01-16 at 10 59 29](https://user-images.githubusercontent.com/701222/212662654-1e979c15-6e51-4d3a-b71f-9780ade7b786.png)

I'd also like to be able to do that with page groups:

![Screenshot 2023-01-16 at 10 59 58](https://user-images.githubusercontent.com/701222/212662747-64e16f47-fb7f-4db2-91e1-f85865b524f9.png)

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
A request similar to this is to enable collapsible page groups that can also be linked to: https://github.com/orgs/GitbookIO/discussions/138

**Reply by taszap** [reply]
Closed in favour of that request
